nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -574,14 +575,6 @@ void NNTrainer::NNTrainerImpl::trainModel() {
     return;
   }
 
-  try {
-    ml_logd("Save_model: %s", model_save_path.c_str());
-    model->save(model_save_path, ml::train::ModelFormat::MODEL_FORMAT_BIN);
-
-  } catch (const std::exception &e) {
-    ml_loge("Error %s, %s", typeid(e).name(), e.what());
-    return;
-  }
```

Comment:
Then, when do you call ```save()``` ?

Reply:
If set as a property, the model is updated at the end of each epoch.
When the stop callback is called, the current epoch is executed and the model is not saved.
```nntrainer``` continues to request data until the end of the current epoch, I double-checked with Choi Hyeongjun.